recipes/argo/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+set -euf
+
+yarn --cwd ui install --frozen-lockfile --ignore-optional --non-interactive
+yarn --cwd ui build
+
+# This could be a conda-package as a build requirement
+# then we would not reed to remove it
+go get bou.ke/staticfiles
```

Comment:
What about licenses for this package?

Reply:
This is not needed, it is used during compile time and then discarded.